libp2p/crypto/crypto.nim

```diff
@@ -222,8 +222,8 @@ proc toBytes*(key: PrivateKey, data: var openarray[byte]): CryptoResult[int] =
   ##
   ## Returns number of bytes (octets) needed to store private key ``key``.
   var msg = initProtoBuffer()
-  msg.write(initProtoField(1, cast[uint64](key.scheme)))
-  msg.write(initProtoField(2, ? key.getRawBytes()))
+  msg.write(1, cast[uint64](key.scheme))
```

Comment:
nice with the write. does this need cast really or would `uint64(key.scheme)` be enough?